src/Server/Game/Game.py

```diff
@@ -22,7 +22,7 @@ def __init__(self):
         self.logger = logging.getLogger('Domi.Game')
         self.__game_started = False
         self.__chose_host = False
-        self.__AI_number = 2
+        self.__AI_number = 5
         self.__human_player_number = 2  # remember to adjust this default with screen's first player's selector's
         self.__first_player_id = None
```

Comment:
That's a lot, I want back 2!

Reply:
Actually my local version has 6 :P 
It's fun if there are many in the party! :)